Kommunicate-io/Kommunicate-Web-SDK
# Added support in hidePostCTA for the case if a list is used as button container

### What do you want to achieve?
- if a list template is used as a container for buttons as shown below, and the list only contains quick replies and hidePostCTA is true, then the buttons should hide along with message itself.

![Screenshot 2021-03-18 at 2 50 11 PM](https://user-images.githubusercontent.com/19753380/111602350-4342f480-87f9-11eb-93e6-78070ceef8c0.png)

### PR Checklist


### How was the code tested?

- create a intent in kompose section with custom payload having list template payload
- the list template payload should only have quick reply buttons
- open your chat widget with hidePostCTA set as true in kommunicate settings
- trigger the intent 
- the list with buttons should appear as shown below 
- upon clicking or upon sending the message the list should hide automatically.
![Screenshot 2021-03-18 at 2 50 11 PM](https://user-images.githubusercontent.com/19753380/111602350-4342f480-87f9-11eb-93e6-78070ceef8c0.png)

Note: The scenario in image below should not happen
![Screenshot 2021-03-18 at 2 51 33 PM](https://user-images.githubusercontent.com/19753380/111602653-9321bb80-87f9-11eb-990a-b8654ca4cfb5.png)

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch
